bsip-0083.md

```diff
@@ -0,0 +1,74 @@
+```
+BSIP: 0083
+Title: Decouple BitAssets from Platform Governance Process
+Authors:    
+   Alfredo Garcia
+   Christopher Sanborn
+   John M. Jones
+   Michel Santos
+   Nathan Hourt
+   Peter Conrad
+   Ryan R. Fox
+Status: Draft
+Type: Protocol
+Obsoletes: BSIP-0016, BSIP-0058, BSIP-0059, BSIP-0075, BSIP-0076
+Created: 2019-10-07
+Discussion: https://github.com/bitshares/bsips/issues/239
+Worker: <TBD>
+```
+
+# Abstract
+This BSIP separates the governance of the technical security of the blockchain from the governance of BitAssets.  BitAssets are a collection of special case SmartCoin which conflate blockchain governance and market effectiveness. To best serve the BitShares protocol, the existing set of BitAssets must be reconfigured to separate the concerns of block production, price feeds, and market operations; that is to say: deterministic consensus, non-deterministic external data, and smart contract settings.
+
+Therefore, this BSIP defines a process for BitAssets management to ensure protocol security remains the singular focus of the block producer role, and give BitAssets the best chance for long-term viability by nominating a new custodian to quickly react to market conditions.  Furthermore, the portion of BTS collateral below the full valuation of a debt asset, as defined by its price feed, that is borrowed from the blockchain, shall not contribute to the voting power of the debtor.
+
+# Motivation
+Block production is an essential component of the BitShares platform itself, and as such should be expected to take an agnostic and neutral view towards the products deployed within the platform. Therefore, this BSIP will define a process for BitAsset management to ensure protocol security remains the singular focus of the block producer role and give BitAssets the best chance for long-term stability by nominating a new custodian to quickly react to market conditions. In order for the distributed consensus mechanism of DPOS to achieve adequate decentralization, it is imperative that block producers maintain strict independence from the politics of individual products deployed on the platform. 
```

Comment:
If your proposed asset owner change for all bitassets passed, the witnesses wouldn't have a singular focus on block production as there are many user created smartcoins which can be fed by witnesses. I have 4 myself fed by several witnesses, configured in a private price feed publisher list.

How was witness decentralization put at threat by the politics of bitassets? By having votes withdrawn if they didn't support rushed BSIPs targeting bitassets? Witnesses are an inherently political role, they have to campaign for their positions in the first place, so politics will always be at the forefront of their minds.

Reply:
We have had block producers being voted out after reliably doing their job for *years*, because they didn't respond to price feed demands from debt holders within *hours*. This is endangering the stability of the blockchain.
By decoupling BitAssets from blockchain governance we hope to prevent witnesses being targeted because of the pricefeed, simply because they will not be feed providers anymore.

Reply:
> 
> because they didn't respond to price feed demands from debt holders within _hours_. 
> By decoupling BitAssets from blockchain governance we hope to prevent witnesses being targeted because of the pricefeed, simply because they will not be feed providers anymore.

I think you need carefully say that “demands from debt holders”. This is vote result.

If your want to prevent witnesses being targeted because of the pricefeed, design a witnees of price feed mechanism.
1.The price feeder didn't need to produce block and just need to provide the price feed, 
2.The witnees of price feed will get the reward from the bitasset which he feed.
3.The witnees of price feed should vote by the stakeholder.
...
I found we learned nothing from the MAKERDAI.

Reply:
>We have had block producers being voted out after reliably doing their job for years, because they didn't respond to price feed demands from debt holders within hours. This is endangering the stability of the blockchain.

Indeed the act of intimidating/coercing witnesses into implementing a BSIP prior to its WP activation (and before the BSIP left draft status) is deplorable ([related bitsharestalk thread](https://bitsharestalk.org/index.php?topic=29659.msg336083#msg336083)).

Should the new asset owner position have been put to public bid, rather than nomination? Given the following most recent public cn-vote rep message on telegram threatening to replace core around the same time this bsip came out raises concerns.

> [09.10.19 11:47 cn-vote]
> The core development team should not forget what they are and what they should do. Do your own job, because there are already many excellent and cheaper teams who have repeatedly sought to compete for the core team.
> BM left, and the bitshares have been running steadily for so many years. Therefore, no one is indispensable. Don't forget who you are and what you should do. Do your job well and everyone will always support you. Otherwise, there will be more suitable teams and people. Thank you all. A job like this high salary and not so busy can be difficult to find. Many excellent teams have been coveted for a long time.

---

> I think you need carefully say that “demands from debt holders”. This is vote result.

It was activated in production prior to any WP activating: https://bitsharestalk.org/index.php?topic=29653.msg335831#msg335831